iModelCore/ECPresentation/Tests/NonPublished/Integration/Content/Content_Filtering.cpp

```diff
@@ -395,6 +395,56 @@ TEST_F(RulesDrivenECPresentationManagerContentTests, DescriptorInstanceFilter_Us
     RulesEngineTestHelpers::ValidateContentSet(bvector<IECInstanceCP>{a2.get()}, *content, true);
     }
 
+/*---------------------------------------------------------------------------------**//**
+* @bsitest
++---------------+---------------+---------------+---------------+---------------+------*/
+DEFINE_SCHEMA(DescriptorInstanceFilter_UsesGuidProperty, R"*(
+    <ECEntityClass typeName="A">
+        <ECProperty propertyName="GuidProp" typeName="binary" extendedTypeName="BeGuid" />
+    </ECEntityClass>
+)*");
+TEST_F(RulesDrivenECPresentationManagerContentTests, DescriptorInstanceFilter_UsesGuidProperty)
+    {
+    ECClassCP classA = GetClass("A");
+    BeGuid instanceGuid1 = BeGuid(true);
+    auto guid1 = instanceGuid1.ToString();
+    IECInstancePtr instance1 = RulesEngineTestHelpers::InsertInstance(s_project->GetECDb(), *classA, [instanceGuid1](IECInstanceR instance)
+        {
+        instance.SetValue("GuidProp", ECValue((Byte*)&instanceGuid1, sizeof(BeGuid)));
+        });
+    BeGuid instanceGuid2 = BeGuid(true);
+    auto guid2 = instanceGuid2.ToString();

```

Comment:
looks like `guid1` and `guid2` are unused

Reply:
removed